index.html

```diff
@@ -12,18 +12,20 @@
   </head>
   <body>
     <div class="container">
-      <h1 class="tittle-brand">Atletismo|
+      <!-- Header -->
+      <h1 class="header-tittle-brand">Atletismo|
         <span>Track & Field</span>
       </h1>
-
+      <!-- navbar -->
       <nav class="navbar sticky-top p-0 navbar-expand-lg bg-light navbar-light">
+
         <!-- Brand -->
         <a class="navbar-brand" href="#"><span>Deporte Rey</span></a>
         <!-- Brand and toggle get grouped for better mobile display -->
-        <button class="navbar-toggler custom-focus" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <button class="navbar-toggler mr-2 custom-focus" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
           <span class="navbar-toggler-icon"></span>
         </button>
-        <!--conainer fluid-->
+        <!-- conainer fluid -->
```

Comment:
` <!-- conainer fluid -->` you made a typo in comment. It should be ` <!-- container fluid -->`

Reply:
this small detail has been solve